### PR TITLE
1747 add functionality to load google satellite as base map

### PR DESCRIPTION
**Pull request by yaguzmang**
This PR adds the satellite view as the base map and recenters to the chosen country in the countries selector.

This PR also includes:

- An update for the URL call of the forest estimations endpoint.
- A fix for the bug crashing the geo page on Safari.
- A small CSS adjustment for the error and loading messages in the statistics panel.

satelliteview3

### CODE REVIEW

File: src/client/pages/Geo/GeoMap/GeoMap.tsx [hunk 1]
```diff
@@ -1,31 +1,66 @@
...
+      // Center and focus on the current country
+      getCountryBounds(countryIso)
+        .then((response) => {
+          if (response?.data) {
+            mapSetUp.panTo(response.data.centroid)
+            mapSetUp.fitBounds(response.data.bounds)
+          }
         })
-      )
+        .catch()
```
Comment (sorja) [inline src/client/pages/Geo/GeoMap/GeoMap.tsx]: Possible to avoid try catch and use await and try/catch?

Comment (minotogna) [reply src/client/pages/Geo/GeoMap/GeoMap.tsx]: at least remove the empty catch ;)

File: src/client/pages/Geo/GeoMap/GeoMap.tsx [hunk 2]
```diff
@@ -35,4 +70,11 @@ const GeoMap: React.FC<Props> = ({ children, center, zoom }) => {
   )
 }
 
+GeoMap.defaultProps = {
+  // Default to Finland's centroid and bounds
```
Comment (minotogna) [inline src/client/pages/Geo/GeoMap/GeoMap.tsx]: I don't think there should a default here. user always lands on a country page, therefore countryIso is there.

File: src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/TreeCoverAreaPanel/TreeCoverAreaPanel.tsx [hunk 3]
```diff
@@ -23,9 +25,7 @@ const TreeCoverAreaPanel: React.FC<Props> = (props: Props) => {
     const sourceName = rowData[0]
     const area = rowData[1]
     const percentage = rowData[2]
-    const formatedArea = Math.round(area)
-      .toString()
-      .replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',') // Comma separator for thousands.
+    const formatedArea = Numbers.format(area, 0)?.toString()
```
Comment (minotogna) [inline src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/TreeCoverAreaPanel/TreeCoverAreaPanel.tsx]: doesn't Number.format already returns string? in that case you can about this toString() ?

File: src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss [hunk 4]
```diff
@@ -12,3 +12,7 @@
     width: 700px;
   }
 }
+
+.geo-map-alt-message {
+  padding: 0 0 1em 1em;
```
Comment (minotogna) [inline src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss]: we don't use em. px is preferred . use $spacing-XXX

File: src/meta/geo/bounds.ts [hunk 5]
```diff
@@ -9,3 +9,14 @@ export interface Bounds {
   countryIso: CountryIso
   data: BoundsData
 }
+
+// Default bounds (Finland)
```
Comment (minotogna) [inline src/meta/geo/bounds.ts]: to remove

**Review by minotogna — COMMENTED** [review]
good pr 👍 just minor details.

File: src/client/pages/Geo/GeoMap/GeoMap.tsx (hunk 1 above)
Comment (yalejandrogg) [reply src/client/pages/Geo/GeoMap/GeoMap.tsx:49]: There is a try-catch in the fetch function, forgot to clean up the catch here 😅

File: src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss (hunk 4 above)
Comment (yalejandrogg) [reply src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss]: Okay, I fixed it here and in the other .scss files from the statistics panel.

File: src/meta/geo/bounds.ts (hunk 5 above)
Comment (yalejandrogg) [reply src/meta/geo/bounds.ts]: Done 👍

File: src/client/pages/Geo/GeoMap/GeoMap.tsx (hunk 1 above)
Comment (yaguzmang) [reply src/client/pages/Geo/GeoMap/GeoMap.tsx]: There is a try-catch in the fetch function, forgot to clean up the catch here 😅. It is fixed now.

File: src/client/pages/Geo/GeoMap/GeoMap.tsx (hunk 2 above)
Comment (yaguzmang) [reply src/client/pages/Geo/GeoMap/GeoMap.tsx]: Done 👍

File: src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/TreeCoverAreaPanel/TreeCoverAreaPanel.tsx (hunk 3 above)
Comment (yaguzmang) [reply src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/TreeCoverAreaPanel/TreeCoverAreaPanel.tsx]: Yes, it is fixed now. Thanks!

File: src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss (hunk 4 above)
Comment (yaguzmang) [reply src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss]: Okay, I fixed it here and in other .scss files from the statistics panel.

File: src/meta/geo/bounds.ts (hunk 5 above)
Comment (yaguzmang) [reply src/meta/geo/bounds.ts]: Done 👍

**Review by minotogna — APPROVED** [review]

File: src/client/pages/Geo/GeoMap/GeoMap.tsx (hunk 1 above)
Comment (minotogna) [reply src/client/pages/Geo/GeoMap/GeoMap.tsx]: this is resolved for me. @sorja resolve the conversation if you are satisfied with the changes please.

**Review by sorja — APPROVED** [review]

**Pull request merged**